src/app/helpers/controller/cms-mixin.js

```diff
@@ -86,23 +85,12 @@ export async function fetchFromCMS(slug, preserveWrapping=false) {
     return preserveWrapping ? data : transformData(data);
 }
 
-function setTitleAndDescription(data) {
-    const meta = data.meta || {};
-    const defaultDescription = data.description ?
-        $.htmlToText(data.description) : '';
-
-    $.setPageTitleAndDescription(
-        meta.seo_title || data.title,
-        meta.search_description || defaultDescription
-    );
-}
-
 export async function fetchPageData({slug, preserveWrapping, setsPageTitleAndDescription=true}) {
     const data = await fetchFromCMS(slug, preserveWrapping);
 
     await loadImages(data);
     if (setsPageTitleAndDescription) {
-        setTitleAndDescription(data);
+        $.setPageTitleAndDescriptionFromBookData(data);
     }
     return data;
```

Comment:
same comment as above, might need to wrap this in useEffect

Reply:
This is not a hook. It is a helper that is used by a hook (and is called from inside `useEffect`)